plugins/modules/ipaautomountlocation.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Authors:
+#   Chris Procter <cprocter@redhat.com>
+#
+# Copyright (C) 2019 Red Hat
```

Comment:
I know you had these automount classes available for a while, but is 2019 correct? If not, please correct the year.